packages/api/src/command/medical/admin-upload-doc.ts

```diff
@@ -0,0 +1,98 @@
+import { v4 as uuidv4 } from "uuid";
+import dayjs from "dayjs";
+import { makeFhirApi } from "../../external/fhir/api/api-factory";
+import { getOrganizationOrFail } from "./organization/get-organization";
+import { getPatientOrFail } from "./patient/get-patient";
+import { Config } from "../../shared/config";
+
+const docContributionUrl = Config.getDocContributionUrl();
+
+export async function createAndUploadDocReference({
```

Comment:
Can we add a TSDoc explicitly stating this is only to be used on `admin` like endpoints/logic, please? It minimizes the risk of someone using this command in the future.